src/Propel/Generator/Builder/Om/ObjectBuilder.php

```diff
@@ -778,7 +780,7 @@ public function addTemporalAccessorComment(&$script, Column $column)
 
         $handleMysqlDate = false;
         if ($this->getPlatform() instanceof MysqlPlatform) {
-            if ($column->getType() === PropelTypes::TIMESTAMP) {
+            if ($column->getType() === PropelTypes::TIMESTAMP || $column->getType() === PropelTypes::TIMESTAMPTZ) {
```

Comment:
this might be unnecessary because of the platform initialize() bits? (Havent checked yet)